language-design.md

```diff
@@ -116,6 +150,6 @@ Briefly describe how you split up the work for this assignment.
 
 **Response**
 
-
+We first met to discuss our overall response to the papers and to choose the quotes we wanted to focus on for the first question. Then, we divided the questions evenly between the two of us and wrote response independently. We reconvened to make edits to each other’s work and continued to revise and expand our answers on google docs. Finally, Andrew formatted the markdown so that Hannah could push the work. 
```

Comment:
That's exactly what we did, too. It worked out well for us, I hope you two had a similar experience. Great minds think alike!